resources/schema.gql

```diff
@@ -958,7 +958,17 @@ type FlowConfigurationCompactionRule {
 
 type FlowConfigurationReset {
 	newHeadHash: Multihash
-	oldHeadHash: Multihash!
+	oldHeadHash: Multihash
+	recursive: Boolean!
+}
```

Comment:
This is used only as a view model, but why do we make a difference? I think it should reflect the propagation mode too.